<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdFilterWidget</class>
 <widget class="QWidget" name="ThresholdFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>Parameter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QGroupBox" name="groupBox_11">
        <property name="title">
         <string>Lower Threshold Pixel Value</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QDoubleSpinBox" name="lowerThresholdSpinbox">
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>-10000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_12">
        <property name="title">
         <string>Upper Threshold Pixel Value</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QDoubleSpinBox" name="upperThresholdSpinbox">
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>-10000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="sdfsdf">
        <property name="title">
         <string>Outside Pixel Value</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QDoubleSpinBox" name="outsideSpinbox">
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>-10000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QPushButton" name="thresholdButton">
     <property name="text">
      <string>Perform</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
